concepts/packages/about.md

```diff
@@ -52,3 +52,19 @@ errors.New("Connection not established")
 // using the errors package located at github.com/pkg/errors by alias
 errs.New("Connection not established")
 ```
+
+Go determines if an item can be called by code in other packages through how it is declared.
+To make a function, type, variable, or constant externally visible (known as [`exported`](https://golang.org/ref/spec#Exported_identifiers)) the name must start with a capital letter.
+This is analogous to how other programming languages like Java that use access modifiers such as `public` and `private`.
+
+```go
+package greeting
+// Hello is a public function (callable from other packages)
+func Hello(name string) string {
+    return "Hello " + name
+}
+// hello is a private function (not callable from other packages)
+func hello(name string) string {
+    return "Hello " + name
+}
+```
```

Comment:
I think it would be good also add a sentence that the same applies for struct field names, I dont think that is obvious from the above

Reply:
Good call for the `about.md`, but should this make it to the `introduction` for the concept or exercise, as it isn't needed to solve them?

Reply:
The introduction files are not only about doing the exercise but also understanding the concept itself. This is a case for something like that.

In about I would write something and show an example. In the introduction files I would just write one sentence about it.